libraries/Microsoft.Bot.Builder/Streaming/StreamingRequestHandler.cs

```diff
@@ -153,9 +150,9 @@ public override async Task<StreamingResponse> ProcessRequestAsync(ReceiveRequest
             var response = new StreamingResponse();
 
             // We accept all POSTs regardless of path, but anything else requires special treatment.
-            if (!string.Equals(request.Verb, StreamingRequest.POST, StringComparison.InvariantCultureIgnoreCase))
+            if (!string.Equals(request?.Verb, StreamingRequest.POST, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Under what conditions is this null?

Reply:
@stevengum can you answer this? This was one of the changes you pushed to this branch.